index.html

```diff
@@ -11,18 +11,7 @@
   <body>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
 
-    <!--
-
-    Hello Camper!
-
-    For now, the test suite only works in Chrome!
-    Please read the README below in the JS Editor before beginning.
-    Feel free to delete this message once you have read it.
-    Good luck and Happy Coding!
-
-    - The freeCodeCamp Team
-
-    -->
+   
     <div class="container">
       <header class="header">
         <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
```

Comment:
* Please consider using classes over id, id's are mostly used with elements that appear only once E.g. header, footer.